app/Http/Requests/Backend/Parking/StoreParkingRequest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Requests\Backend\Parking;
+
+use Illuminate\Validation\Rule;
+use Illuminate\Foundation\Http\FormRequest;
+
+/**
+ * Class StoreParkingRequest.
+ */
+class StoreParkingRequest extends FormRequest
+{
+	/**
+	 * Determine if the user is authorized to make this request.
+	 *
+	 * @return bool
+	 */
+	public function authorize()
+	{
+		return $this->user()->isAdmin();
+	}
+
+	/**
+	 * Get the validation rules that apply to the request.
+	 *
+	 * @return array
+	 */
+	public function rules()
+	{
+		return [
+			'name'     => ['required', 'max:191', Rule::unique('parking')],
+			'status'     => ['boolean'],
+			'capacity'     => ['required', 'max:191', 'integer'],
+			'price'     => ['required', 'max:191', 'integer'],
+			'latitude'     => ['required', 'regex:/^[-]?(([0-8]?[0-9])\.(\d+))|(90(\.0+)?)$/'],
+			'longitude'     => ['required', 'regex:/^[-]?(([0-8]?[0-9])\.(\d+))|(90(\.0+)?)$/'],
```

Comment:
'The longitude format is invalid.' response on inputting valid value for longitude. The regex for longitude needs to be updated.
Latitudes range from -90 to 90.
Longitudes range from -180 to 180.

Reply:
let's skip this issues and let's focus on other tasks.